completion_aggregator/fields.py

```diff
@@ -0,0 +1,87 @@
+"""
```

Comment:
I think this can go away now, with the update to edx-opaque-keys.

Reply:
Oh yeah.  I haven't updated this with all the recently landed changes.  I'll do that, then ping you for a re-review.